README.md

```diff
@@ -151,6 +196,10 @@ module purge
 module load nextflow singularity
 ```
 
+
+
+
+
 To run this workflow on NCI Gadi HPC, you can excecute the script `scripts/run_pipeline_on_gadi_script.sh` by first entering the following details in the PBS header of the script:
 - project code
 - Resource-related details  
```

Comment:
Also going to create another issue for this to be addressed in the benchmarking PR. The mem and ncpu resources consumed by the head nextflow job submitted using your pbs script will be standard for all runs and we should just provide them.